src/Html.php

```diff
@@ -4683,6 +4684,7 @@ public static function jsAjaxDropdown($name, $field_id, $url, $params = [])
         $valuename = $params['valuename'];
         $on_change = $params["on_change"];
         $placeholder = $params['placeholder'] ?? '';
+        $multiple = $params['multiple'] ?? '';
```

Comment:
This param is defined in `$default_options`, so it is always defined. Thus `''` default value is not correct.
```suggestion
        $multiple = $params['multiple'];
```